tests/autoblocks/test_tracer.py

```diff
@@ -448,3 +451,90 @@ def test_tracer_start_span(*args, **kwargs):
 
     assert tracer._properties.get("span_id") is None
     assert tracer._properties.get("parent_span_id") is None
+
+
+def test_tracer_prod_evaluations(httpx_mock):
+    test_evaluation_id = uuid.uuid4()
+
+    class MyEvaluator(BaseEventEvaluator):
+        id = "my-evaluator"
+
+        def evaluate_event(self, event: TracerEvent) -> EventEvaluation:
+            return EventEvaluation(
+                evaluator_external_id=self.id,
```

Comment:
this is also the same thing i was getting at in another PR, where we shouldn't leak internal implementation details to schemas used by end users